src/index.ts

```diff
@@ -23,10 +22,8 @@ export {
     type SiteResult,
 } from "./stores/response";
 export { getAst } from "./helpers/ast-transformer";
-
-// export temporarily, this will hopefully be moved to CCP explorer before 0.6.0 release
-export { translateAstToCql } from "./cql-translator-service/ast-to-cql-translator";
 export { buildLibrary, buildMeasure } from "./helpers/cql-measure";
+export { getSelectedSites, getHumanReadableQuery } from "./stores/datarequests";
```

Comment:
Just a thought, should `getHumanReadableQuery` be in an other file? Since it is used in the QueryExplainButton